agent/src/main/java/io/perfmark/agent/PerfMarkTransformer.java

```diff
@@ -151,7 +151,7 @@ public void visitSource(String sourceFileName, String debug) {
     public MethodVisitor visitMethod(
         int access, String name, String descriptor, String signature, String[] exceptions) {
       clinitSeen = clinitSeen || name.equals("<clinit>");
-      return new PerfMarkMethodVisitor(
+      return new PerfMarkClassReader.PerfMarkMethodVisitor(
```

Comment:
This fixes the new errorprone warning.
https://gist.github.com/suztomo/a57c6e537c8f35795e365bbf71559ead#file-gistfile1-txt-L24